### PR TITLE
[ADMINAPI-442] - Minor updates to Admin API endpoints

### PR DESCRIPTION
**Pull request by jagudelo-gap**

### CODE REVIEW

**Review by jasonh-edfi — APPROVED** [review]
Updates reviewed and approved.

File: Application/EdFi.Ods.AdminApi/Infrastructure/Database/Commands/AddApplicationCommand.cs [hunk 1]
```diff
@@ -27,8 +27,8 @@ public AddApplicationCommand(IUsersContext usersContext, InstanceContext instanc
 
     public AddApplicationResult Execute(IAddApplicationModel applicationModel)
     {
-        var profile = applicationModel.ProfileId.HasValue
-            ? _usersContext.Profiles.SingleOrDefault(p => p.ProfileId == applicationModel.ProfileId.Value)
+        var profiles = applicationModel.ProfileIds != null
+            ? _usersContext.Profiles.Where(p => applicationModel.ProfileIds!.Contains(p.ProfileId))
```
Comment (CSR2017) [inline Application/EdFi.Ods.AdminApi/Infrastructure/Database/Commands/AddApplicationCommand.cs:31]: @jagudelo-gap 
I see Profile id list is nullable type. Adding "!" will convince the compiler. But there is a chance for getting a null value at runtime. So please make sure to check null.

Comment (jagudelo-gap) [reply Application/EdFi.Ods.AdminApi/Infrastructure/Database/Commands/AddApplicationCommand.cs:31]: @CSR2017 
The check for null is in line 30

File: Application/EdFi.Ods.AdminApi/Infrastructure/AutoMapper/AdminApiMappingProfile.cs [hunk 2]
```diff
@@ -31,29 +31,29 @@ public AdminApiMappingProfile()
...
             .ForMember(dst => dst.ContactName, opt => opt.MapFrom(src => src.ContactName()))
             .ForMember(dst => dst.ContactEmailAddress, opt => opt.MapFrom(src => src.ContactEmail()))
             .ForMember(dst => dst.NamespacePrefixes, opt => opt.MapFrom(src => src.VendorNamespacePrefixes.ToCommaSeparated()));
 
-        CreateMap<EdFi.Admin.DataAccess.Models.Application, ApplicationModel>()
-            .ForMember(dst => dst.EducationOrganizationId, opt => opt.MapFrom(src => src.EducationOrganizationId()))
-            .ForMember(dst => dst.ProfileName, opt => opt.MapFrom(src => src.ProfileName()))
+        CreateMap<Admin.DataAccess.Models.Application, ApplicationModel>()
+            .ForMember(dst => dst.Id, opt => opt.MapFrom(src => src.ApplicationId))
+            .ForMember(dst => dst.EducationOrganizationIds, opt => opt.MapFrom(src => src.EducationOrganizationId()))
```
[the remark this answers is not in the document]
Comment (jagudelo-gap) [reply Application/EdFi.Ods.AdminApi/Infrastructure/AutoMapper/AdminApiMappingProfile.cs:42]: @CSR2017 
No, we are returning a list, maybe we should rename the extension EducationOrganizationId()

File: Application/EdFi.Ods.AdminApi/Infrastructure/Database/Commands/AddApplicationCommand.cs (hunk 1 above)
Comment (CSR2017) [reply Application/EdFi.Ods.AdminApi/Infrastructure/Database/Commands/AddApplicationCommand.cs:31]: Sorry Juan. Wow, how did I miss it !! THanks

**Review by CSR2017 — APPROVED** [review]

**Pull request merged**